### DISCUSSION TITLE
How to adjust the font spacing？

### DISCUSSION THREAD

**Discussion by zhenzi0322** [fontforge/fontforge#5177] [Q&A]
How to adjust the font spacing？

**Comment by samiel60** [comment]
Do you mean font (left and right) bearing? I don't see bugs here in FF!

**Reply by zhenzi0322** [reply]
yes.

**Comment by samiel60** [comment]
Can you describe the bugs you think you found?
In the "Metrics" menu you find all the fields you should need to set the (L and R) bearing. Once this is done, for glyphs couples where bearing is not enough you have to work with kerning.

**Reply by zhenzi0322** [reply]
Is there no way to make adjustments in the code?
